content/events/infra-3.0-brussels.md

```diff
@@ -0,0 +1,41 @@
+---
+eventName: "Infra 3.0 - Brussels DePIN Deep Dive JUL 9" #the name of your track or event and its mandatory
+
+eventDescription: "Join us and dive deep into the latest topics and research in the realm of DePIN: cloud services, multi-party computation, decentralized RPC and others.
+
+​Enjoy keynotes and panel discussions, connect with DePIN leaders, researchers, developers, and enthusiasts during EthCC in Brussels." #short description of your track or event limiting to 100-150 characters
+
+#Topics will show up as labels on the event card.Only a maximum of 4 tags will be displayed on the event card. Some references for topics - Blockchain, Web3, Cryptocurrency, Tech Talks, Workshop, etc.
+
+eventTopic: 
+  - "Web3"
+  - "DePIN"
+
+eventType: "Conference"
+location: "Brussels, Belgium" 
+venueName: "Maison Grand-Place" #the event venue name (will show up on the event card) or just leave it blank
+
+venueAddress: "Grand Place 19, 1000 Bruxelles, Belgium" #the event venue address (will show up on a map) or just leave it blank
+
+venueMapsLink: "[<Google_Map_link_for_the_venue>](https://www.google.com/maps/place/Grote+Markt+Huis/@50.8464357,4.3532306,17z/data=!3m1!4b1!4m6!3m5!1s0x47c3c47f38d8a66d:0xd3236931131288b3!8m2!3d50.8464357!4d4.3532306!16s%2Fg%2F1222z5lb?entry=ttu)" #the event venue Map link (will show up on a map) or just leave it blank
```

Comment:
@laeccles Please confirm if this link is correct, as this points to a different location 'Grote Markt Huis' that is different to the one given above in the Locations field.

Reply:
@WinsCodes I believe this points to the correct location -  Please keep the link
<img width="209" alt="Screen Shot 2024-07-02 at 8 36 10 AM" src="https://github.com/memser-spaceport/pln-events/assets/112510736/f9df6632-f0b9-483f-9e80-aefa57bc0e7a">